packages/p2p/src/stores/general-store.js

```diff
@@ -746,7 +746,7 @@ export default class GeneralStore extends BaseStore {
             const { list } = p2p_order_list;
             // it's an array of orders from p2p_order_list
             this.handleNotifications(order_store.orders, list);
-            list.forEach(order => order_store.syncOrder(order));
+            list?.forEach(order => order_store.syncOrder(order));
```

Comment:
@adrienne-deriv .. i have a doubt.. i think this issue was caused due to the destructuring in line 742.. 
     `const { p2p_order_list = [], p2p_order_info = {} } = order_response;`
     
if p2p_order_list is given empty array as default value, then won't if of empty array will always be true and the else condition for that be never satisified?

Reply:
Actually yeahh the first condition would always be true even if `p2p_order_list` is undefined from order_response 🤣 
I'm checking the [pull request](https://github.com/binary-com/deriv-app/pull/8274/files) that added the default assignments, I'm not so sure why this was added

Reply:
i think changing 
`const { p2p_order_list = [], p2p_order_info = {} } = order_response; ` to 
`const { p2p_order_list, p2p_order_info } = order_response ?? {};`
 would fix the issue. since we have and if and else condition check when it is undefined it wont break anything.

Reply:
Yeah I think this change is appropriate, since it doesn't cause an issue where the first/second condition will always pass even if its not in `order_response`